### DISCUSSION TITLE
Add NFTables firewall interface to IPBan?

### DISCUSSION THREAD

**Discussion by dcalnetstock** [DigitalRuby/IPBan#296] [General]
Hi, we run Debian and Windows in our server fleet, and enjoy IPBan on our windows servers. We'd love to streamline our IPBanning infrastructure (and deprecate fail2ban) on linux. However, the binary that runs IPBan only seems to support iptables. Is there an appetite in the community to integrate IPBan with Linux NFTables?

**Comment by jjxtra** [comment]
FirewallD is also supported (and is the default Linux implementation as of 2.0.0), which should have an nftabels backend.

**Reply by dcalnetstock** [reply]
got it, thank you. That makes sense.
